pkgs/development/python-modules/hpack/default.nix

```diff
@@ -15,7 +15,7 @@ buildPythonPackage rec {
 
   buildInputs = [ glibcLocales ];
```

Comment:
```suggestion
  nativeBuildInputs = [ glibcLocales ];
```

Reply:
Is this really still needed for this package?

Reply:
It seems it is for python36.

Reply:
Ahh, I see the pr title includes `python36Packages` so that's where the failure is.

Perhaps it should just be conditioned for that interpreter version.

Reply:
I'll add a comment that it's for 3.6.